fluent-react/src/localized.ts

```diff
@@ -51,8 +45,8 @@ function toArguments(props) {
  *  translation is available.  It also makes it easy to grep for strings in the
  *  source code.
  */
-function Localized(props) {
-  const { id, attrs, children: child = null } = props;
+export function Localized(props: LocalizedProps): ReactElement {
```

Comment:
Why not loosen the return type to [`ReactNode`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L203) instead, which includes `ReactElement` but also allows `child` to be returned without a wrapping call.

Reply:
I did that initially, but I ran into https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18912. See https://github.com/projectfluent/fluent.js/pull/458#issuecomment-595875336.